### ISSUE TITLE
'ReferenceError: Quagga is not defined' when using script 'dist/quagga.js'

### ISSUE THREAD

**Issue by WardLootens** [ericblade/quagga2#113]
I'm trying this:

>You can simply include dist/quagga.min.js in your project and you are ready to go. The script exposes >the library on the global namespace under Quagga.

However, I cannot seem to get this working. I get this error in the browser console:
```
ReferenceError: Quagga is not defined
```

I generated the `dist/quagga.js` file by running `npm run build`

**edit**
I can get it working by adding
```
const Quagga = window.quagga.default;
```

**Comment by WardLootens** [comment]
My guess is that this is broken since d2387aab (`upgrade to webpack 4`)

**Comment by WardLootens** [comment]
Seems to be fixed by using this in `webpack.config.js`

```
    output: {
        path: __dirname + '/../dist',
        publicPath: '/',
        libraryTarget: 'umd',
        libraryExport: 'default',
        library: 'Quagga',
        filename: 'quagga.js',
    },
```

I will submit a pull request

**Comment by WardLootens** [comment]
Fixed with PR #114

**Comment by ericblade** [comment]
Thank you, and again, my apologies for not catching it -- I thought the tests that existed for all of this were a bit more ... robust? than they really are.
